modules/ROOT/pages/database-administration/aliases/manage-aliases-standard-databases.adoc

```diff
@@ -0,0 +1,842 @@
+| name                          | composite | database               | location | url                       | user    | driver                                                   | properties |
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| "remote-with-driver-settings" | NULL      | "northwind-graph-2020" | "remote" | "neo4j+s://location:7687" | "alice" | {connection_pool_max_size: 10, connection_timeout: PT1M} | {}         |
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+----
+
+
+==== Set properties for remote database aliases
+
+Just as the local database aliases, the remote database aliases can be given properties.
+These properties can then be used in queries with the xref::database-administration/composite-databases/querying-composite-databases.adoc#composite-databases-queries-built-in-functions[`graph.propertiesByName()` function].
+
+.Query
+[source, cypher]
+----
+CREATE ALIAS `remote-northwind-2021` FOR DATABASE `northwind-graph-2021` AT 'neo4j+s://location:7687'
+USER alice PASSWORD 'password'
+PROPERTIES { newestNorthwind: false, index: 6 }
+----
+
+The properties are then shown in the `SHOW ALIASES FOR DATABASE YIELD ...` command.
+
+.Query
+[source, cypher]
+----
+SHOW ALIAS `remote-northwind-2021` FOR DATABASE YIELD name, properties
+----
+
+.Result
+[role="queryresult"]
+----
++--------------------------------------------------------------+
+| name                    | properties                         |
++--------------------------------------------------------------+
+| "remote-northwind-2021" | {index: 6, newestnorthwind: FALSE} |
++--------------------------------------------------------------+
+----
+
+[NOTE]
+====
+For more information on  local and remote database aliases in a composite database, see xref::database-administration/composite-databases/manage-composite-databases.adoc#create-composite-database-alias[Create database aliases in composite databases].
+====
+
+[[alias-management-alter-database-alias]]
+== Alter database aliases
+
+////
+CREATE ALIAS garden.flowers FOR DATABASE `perennial-flowers`;
+CREATE ALIAS `northwind` FOR DATABASE `northwind-graph-2020`; // created in the replace alias example
+CREATE ALIAS `remote-northwind` FOR DATABASE `northwind-graph-2020` AT "neo4j+s://location:7687" USER alice PASSWORD 'password';
+CREATE ALIAS `remote-with-driver-settings` FOR DATABASE `northwind-graph-2020` AT "neo4j+s://location:7687" USER alice PASSWORD 'password'
+DRIVER {
+    connection_timeout: duration({ minutes: 1 }),
+    connection_pool_max_size: 10
+    };
+CREATE ALIAS garden.trees FOR DATABASE trees AT 'neo4j+s://location:7687' USER alice PASSWORD 'password'
+////
```

Comment:
I wonder if this test setup will be an issue now as it's supposed to capture what is given in the create section and I think that the tests are setup to run everything in the file from top to bottom (unless told to skip) and these then would conflict? or if it needs the [query] blocks for the testing, is there any reason to have these set-up sections at all?

(I think they came from how nested our test generated docs were and the alter section not seeing the create queries for some reason and therefore had it's own setup to duplicate what was supposed to have happened but that it didn't see if I remember correctly)

Reply:
Yes, you are correct. Once we have these pages tested, we will have to go through all the setup queries. Will leave for now to avoid confusion.